lib/dealership.rb

```diff
@@ -0,0 +1,48 @@
+class Dealership
+  attr_accessor :inventory, :inventory_count, :details
+
+  def initialize(dealership, address)
+    @dealership = dealership
+    @address = address
+    @inventory = []
+  end
+
+  def add_car(car)
+    @inventory.push(car)
+  end
+
+  def inventory_count
+    @inventory.size
+  end
+
+  def cars_by_make(make) # try to refactor without if statement if time.
+    car_match = []
+    @inventory.find_all do |car|
+      if car.make == make
+        car_match.push(car)
+      end
+    end
+  end
```

Comment:
I like that you left yourself comments here! 

Some ways to improve this code would be utilizing the `find_all`. You are super close:

```
def cars_by_make(make)
  @inventory.find_all do |car|
    car.make == make
  end
end
```

Your approach is closer to how you might use `.each` rather than `.find_all` since you aren't using the return from `find_all`.